### ISSUE TITLE
Typo in states/virtualenv_mod.py

### ISSUE THREAD

**Issue by jamesp9** [saltstack/salt#33565]
### Description of Issue/Question
Line 192 in "/usr/lib/python2.7/dist-packages/salt/states/virtualenv_mod.py"
`formar(err)` should be
`format(err)`

### Steps to Reproduce Issue
Use the `virtualenv.managed` state to try and create a virtualenv environment.

```
----------
          ID: create_my_virtualenv
    Function: virtualenv.managed
        Name: venv
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "/usr/lib/python2.7/dist-packages/salt/state.py", line 1703, in call
                  **cdata['kwargs'])
                File "/usr/lib/python2.7/dist-packages/salt/loader.py", line 1649, in wrapper
                  return f(*args, **kwargs)
                File "/usr/lib/python2.7/dist-packages/salt/states/virtualenv_mod.py", line 192, in managed
                  ret['comment'] = 'Failed to create virtualenv: {0}'.formar(err)
              AttributeError: 'str' object has no attribute 'formar'
```

### Versions Report
```
# salt --versions-report
Salt Version:
           Salt: 2016.3.0

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.11+ (default, Apr 17 2016, 14:00:29)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
        machine: x86_64
        release: 4.4.0-21-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by Ch3LL** [comment]
@jamesp9 thanks for pointing that out and what looks like the fix. :+1:

**Comment by cachedout** [comment]
We already merged a fix for this in #33565
